enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -6,11 +6,11 @@ import { aceEditorStyles } from "metabase/query_builder/components/NativeQueryEd
 import { saveDomImageStyles } from "metabase/visualizations/lib/save-chart-image";
 
 export const SdkContentWrapper = styled.div<{ font: string }>`
-  --default-font-family: "${({ font }) => font}";
-  --color-brand: ${color("brand")};
-  --color-brand-alpha-04: ${alpha("brand", 0.04)};
-  --color-brand-alpha-88: ${alpha("brand", 0.88)};
-  --color-focus: ${color("focus")};
+  --mb-default-font-family: "${({ font }) => font}";
+  --mb-color-brand: ${color("brand")};
+  --mb-color-brand-alpha-04: ${alpha("brand", 0.04)};
+  --mb-color-brand-alpha-88: ${alpha("brand", 0.88)};
+  --mb-color-focus: ${color("focus")};
```

Comment:
Basically, these are only 5 CSS custom properties that we exposed in HTML. Other custom properties in `:root` is processed by PostCSS and [are removed ](https://github.com/metabase/metabase/blob/8ff9805d9d0cfe360299a7f3d4e2b86bcd80b48d/postcss.config.js#L17-L19)